ote/src/cljs/ote/views/error_landing.cljs

```diff
@@ -0,0 +1,15 @@
+(ns ote.views.error-landing
```

Comment:
Consider moving to own package like errors or somethin. ote.views.errors.error-landing

Reply:
Ok...what role and purpose that package would serve, in addition to the error landing page?

Reply:
We have quite many files in the views folder/package. We should split them to sub folders to help us navigate and find relevant files.

e.g. we have 2 at least 5 files that relates to transport-service and they are in alphabetical order. If they were in single sub folder finding them would be faster.

And from this point of view I'm expecting that we will have more than one error page in a few months so I would suggest that we add one folder for those :)

Reply:
As discussed, design is to have just one reused error_landing view and file. Nevertheless I moved this to new package.